src/go/k8s/helm-chart/charts/redpanda-operator/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{/*
+Copyright 2020 Vectorized, Inc.
+
+Use of this software is governed by the Business Source License
+included in the file licenses/BSL.md
+
+As of the Change Date specified in that file, in accordance with
+the Business Source License, use of this software will be governed
+by the Apache License, Version 2.0
+*/}}
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "redpanda-operator.name" . }}-metrics-monitor
+  labels:
+{{ include "redpanda-operator.labels" . | indent 4 }}
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    path: /metrics
+    port: https
```

Comment:
i don't ge3t why the port and scheme are the same

Reply:
The `port` is in the Prometheus operator must be set to the name of the service port. In our case https://github.com/vectorizedio/redpanda/blob/bb522ea6e70f8243f6fccb735df5d975b3a626fb/src/go/k8s/config/rbac/auth_proxy_service.yaml#L10

REF:
https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint

| Field | Description | Scheme | Required |
| ----- | ----------- | ------ | -------- |
| port | Name of the service port this endpoint refers to. Mutually exclusive with targetPort. | string | false |

Reply:
got it. thanks for explaining. crazy that you have a `name` that is called `port` for the `protocol` but also a `scheme` that is the actual protocol. what gives.

Reply:
We can change the name both in `auth_proxy_service.yaml` and `servicemonitor.yaml`. 

I want to make sure that we don't mix Redpanda admin listener metrics with Redpanda operator.
The port number comes from `kube-rbac-proxy` that guard Redpanda operator.